test/blackbox-tests/test-cases/virtual-libraries/same-directory.t

```diff
@@ -0,0 +1,18 @@
+Test that includes vlib and implementations all in the same folder.
+
+  $ echo "(lang dune 3.6)" > dune-project
+
+  $ touch empty.mli
+  $ cat >dune <<EOF
+  > (library
+  >  (name impl_one)
+  >  (implements vlib))
+  > (library
+  >  (name vlib)
+  >  (virtual_modules empty))
+  > EOF
+  $ dune build
+  File "dune", line 1, characters 0-0:
```

Comment:
The location should come from the stanza rather than the dune file. I suppose the lookup_vlib function should take it as an argument then.